src/fstests/fstests.kernelci.conf

```diff
@@ -0,0 +1,3 @@
+[fstests_runner]
```

Comment:
I think you might need to add other settings such as the API URL etc.  Basically everything except optional things like `--verbose` and credentials such as the API token.